cypress-smoketests/integration/trial-sessions/trial-session-working-copy.spec.js

```diff
@@ -107,6 +113,8 @@ describe('Petitioner', () => {
   });
 });
 
+// eslint-disable-next-line no-unused-vars
+let judgeUserId;
```

Comment:
Is this needed?

Reply:
It will be when the tests are all re-enabled!